components/scream/cmake/machine-files/quartz.cmake

```diff
@@ -0,0 +1,7 @@
+set(CMAKE_CXX_FLAGS "-w" CACHE STRING "")
+set(KOKKOS_ENABLE_DEBUG FALSE CACHE BOOL "")
+set(KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION FALSE CACHE BOOL "")
+set(KOKKOS_ENABLE_CUDA FALSE CACHE BOOL "")
```

Comment:
If quarts and syrah are openMP machines, you should not use the CUDA machine .cmake files as a template. You can use blake as a template but be sure to get the architecture right.

Reply:
this latest commit should fix this, can you take a look?